src/webapi/tests/integ.rs

```diff
@@ -0,0 +1,60 @@
+use once_cell::sync::Lazy;
+use std::{net::TcpListener, str::FromStr};
+use tracing::Level;
+use webapi::{
+    startup::run,
+    telemetry::{get_subscriber, init_subscriber},
+};
+
+// Ensure that the `tracing` stack is only initialised once using `once_cell`
+static TRACING: Lazy<()> = Lazy::new(|| {
+    let subscriber_name = "test".to_string();
+    if let Ok(env_var) = std::env::var("TEST_LOG") {
+        let level = tracing::Level::from_str(&env_var)
+            .unwrap_or(Level::INFO)
+            .as_str();
```

Comment:
Why not just pass in a `Level`, instead of parsing a string only to convert it back into a string?

Reply:
fair enough, changed to `tracing::log::Level`